packages/auth-client/src/types/engine.ts

```diff
@@ -85,6 +85,7 @@ export declare namespace AuthEngineTypes {
 
   interface PendingRequest {
     id: number;
+    topic: string;
```

Comment:
Key should be explicitly `pairingTopic`, see spec here: https://github.com/WalletConnect/walletconnect-docs/pull/601/files#diff-cfea69f999866353f919ffdf5ae18781bb7009beeb583d79a84aa1fa289a5e59R58